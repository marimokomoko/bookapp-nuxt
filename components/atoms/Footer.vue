<template>
  <v-footer :absolute="!fixed" app>
    <span>&copy; {{ new Date().getFullYear() }}</span>
  </v-footer>
</template>

<script lang="ts">
import { Vue, Component } from 'nuxt-property-decorator'
@Component({})
export default class Footer extends Vue {
  private fixed = false
}
</script>

<style>
</style>