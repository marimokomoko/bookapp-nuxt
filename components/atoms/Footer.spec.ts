import { mount, RouterLinkStub } from '@vue/test-utils'
import Footer from '@/components/atoms/Footer.vue'
describe('画面レイアウト', ()=>{
    it('スナップショットテスト', ()=>{
        const wrapper = mount(Footer, { stubs: { NuxtLink: RouterLinkStub }})
        expect(wrapper.html()).toMatchSnapshot()
    })
})