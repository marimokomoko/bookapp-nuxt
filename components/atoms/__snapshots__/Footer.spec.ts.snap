// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `"<v-footer absolute=\\"true\\" app=\\"\\" color=\\"purple lighten-3\\"><span>© 2022</span></v-footer>"`;
